3MI0800331/Task2.cpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+using namespace std;
+
+
+struct orderedPair {
+	int first;
+	int second;
+};
+
+void print(const orderedPair& arg) {
+		cout << arg.first << " " << arg.second;
```

Comment:
Adding a user prompt to suggest what's expected from the user is always a good idea :P